build.proj

```diff
@@ -5,17 +5,16 @@
   <Target Name="Build" DependsOnTargets="PrepareOutput;InitBuild">
     <Message Text="Build Environment: $(Platform) $(Configuration) $(TargetOS) $(TargetRid)" />
 
-    <ItemGroup>
-      <_RootProject Include="$(ProjectDir)targets/repository.proj" Condition="'%(Repository.Identity)' == '$(DefaultRepository)'">
-        <AdditionalProperties>RepositoryName=%(Repository.Identity);PathToRepo=%(Repository.PathToRepo);RepositoryOrganization=%(Repository.Organization);RepositoryBranch=%(Repository.Branch)</AdditionalProperties>
-      </_RootProject>
-    </ItemGroup>
+    <MSBuild Projects="repos\$(RootRepo).proj" Targets="Build" BuildInParallel="$(BuildInParallel)" />
+  </Target>
 
-    <MSBuild Projects="@(_RootProject)" Targets="BuildRepositoryAndDependencies" BuildInParallel="$(BuildInParallel)" />
+  <Target Name="ApplyPatches">
```

Comment:
Shouldn this target condition on `'$(RootRepo)' != ''`?

Reply:
RootRepo is always defaulted so someone would need to explicitly pass empty for it to be empty and if they did they would get an error saying cannot find project. If I were to add a condition I would also want to add an error saying the project doesn't exist and that is basically what the msbuild task does for me if it doesn't exist.